From patchwork Wed May  4 21:15:34 2016
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [U-Boot,6/6] Pine64: rename defconfig
From: Andre Przywara <andre.przywara@arm.com>
X-Patchwork-Id: 618681
Message-Id: <1462396534-32390-7-git-send-email-andre.przywara@arm.com>
To: Hans de Goede <hdegoede@redhat.com>,
	Ian Campbell <ijc@hellion.org.uk>
Cc: Andre Przywara <andre.przywara@arm.com>, u-boot@lists.denx.de
Date: Wed,  4 May 2016 22:15:34 +0100

Rename the defconfig file for the Pine64 from pine64_plus_defconfig to
pine64_defconfig.
The differences between the two versions (more RAM and a different
Ethernet PHY) don't justify two board versions, so lets stick with the
generic name and try to differentiate between the versions at runtime
if this is needed later.

Signed-off-by: Andre Przywara <andre.przywara@arm.com>
---
 configs/pine64_defconfig      | 20 ++++++++++++++++++++
 configs/pine64_plus_defconfig | 20 --------------------
 2 files changed, 20 insertions(+), 20 deletions(-)
 create mode 100644 configs/pine64_defconfig
 delete mode 100644 configs/pine64_plus_defconfig

diff --git a/configs/pine64_defconfig b/configs/pine64_defconfig
new file mode 100644
index 0000000..0977334
--- /dev/null
+++ b/configs/pine64_defconfig
@@ -0,0 +1,20 @@
+CONFIG_ARM=y
+CONFIG_ARCH_SUNXI=y
+CONFIG_MACH_SUN50I=y
+CONFIG_DRAM_CLK=672
+CONFIG_DRAM_ZQ=3881915
+# CONFIG_VIDEO is not set
+CONFIG_DEFAULT_DEVICE_TREE="sun50i-a64-pine64-plus"
+# CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
+CONFIG_HUSH_PARSER=y
+# CONFIG_CMD_IMLS is not set
+# CONFIG_CMD_FLASH is not set
+CONFIG_CMD_MMC=y
+# CONFIG_CMD_FPGA is not set
+CONFIG_CMD_DHCP=y
+CONFIG_CMD_MII=y
+CONFIG_CMD_PING=y
+CONFIG_CMD_EXT2=y
+CONFIG_CMD_EXT4=y
+CONFIG_CMD_FAT=y
+CONFIG_CMD_FS_GENERIC=y
diff --git a/configs/pine64_plus_defconfig b/configs/pine64_plus_defconfig
deleted file mode 100644
index 0977334..0000000
--- a/configs/pine64_plus_defconfig
+++ /dev/null
@@ -1,20 +0,0 @@
-CONFIG_ARM=y
-CONFIG_ARCH_SUNXI=y
-CONFIG_MACH_SUN50I=y
-CONFIG_DRAM_CLK=672
-CONFIG_DRAM_ZQ=3881915
-# CONFIG_VIDEO is not set
-CONFIG_DEFAULT_DEVICE_TREE="sun50i-a64-pine64-plus"
-# CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
-CONFIG_HUSH_PARSER=y
-# CONFIG_CMD_IMLS is not set
-# CONFIG_CMD_FLASH is not set
-CONFIG_CMD_MMC=y
-# CONFIG_CMD_FPGA is not set
-CONFIG_CMD_DHCP=y
-CONFIG_CMD_MII=y
-CONFIG_CMD_PING=y
-CONFIG_CMD_EXT2=y
-CONFIG_CMD_EXT4=y
-CONFIG_CMD_FAT=y
-CONFIG_CMD_FS_GENERIC=y
